CONTRIBUTING.md

```diff
@@ -24,4 +24,11 @@ We use the
 and review all changes we merge to master.
 
 We appreciate any contributions, so please use the [Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow)
-and send us `Merge Requests`
+and send us `Merge Requests`. 
+
+In order to pass the checks, commit with signoff option
+
+```shell
+git commit --signoff
```

Comment:
Hmpf, now I realized that this repo is not yet documenting what we described e.g. here about the signed-off:

https://github.com/siemens/efibootguard/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11

Should be added as well.

Reply:
Added also the Developer's Certificate of Origin 1.1